### ISSUE TITLE
Can you please add an option to change the resolution of the iframe ?

### ISSUE THREAD

**Issue by 0xCD0** [dimitrikarpov/react-emulatorjs#6]
Thank you for your project.
Looking at the return items in the EmulatorJS.tsx file, the height and width are fixed, and I would really appreciate it if you could pass them as props to adjust the resolution.

![image](https://github.com/dimitrikarpov/react-emulatorjs/assets/4960838/0de6c218-5aeb-4756-b74e-e2680034d2d1)

Thank You.

**Comment by dimitrikarpov** [comment]
Hello. Thanks for feedback.

Yes, these params are fixed. But they are just initial params. Sure we can change whem on start, but we could not update emulator dimensions while he running. Which is not the best option. I will research how to manage width and height with parent's div params. So it would take all available space in parent's container dynamically. 

If it would be a complicated approach we can fallback to passing dimensions as props.

**Comment by dimitrikarpov** [comment]
Size props `width` and `height` has been added. Just update `react-emulatorjs` package. For now You can set fixed size, not responsive. Emulatorjs team working on "custom aspect ratio" feature. And it theoretically can broke my responsive styles.

Usage example:
```tsx
<EmulatorJS
  EJS_core = "fceumm"
  EJS_gameUrl = "roms/Spacegulls-1.1.zip"
  EJS_pathtodata = "https://cdn.emulatorjs.org/4.0.8/data"
  EJS_startOnLoaded
  width = 800
  height = 600
/>
```
default values: 640x480

**Comment by 0xCD0** [comment]
Thank you so much for the update.
I will be try it soon.  Have a nice day !  👍🏻

**Comment by dimitrikarpov** [comment]
I found a bug. If start emulator without EJS_startOnLoaded option Start button become unreachable. Will try to fix it tommorow.

**Comment by dimitrikarpov** [comment]
Fixed in 2.2.5 🤓
